src/orchestrator-process/src/lib.rs

```diff
@@ -342,6 +342,9 @@ impl NamespacedOrchestrator for NamespacedProcessOrchestrator {
             labels,
             availability_zone: _,
             anti_affinity: _,
+            disk,
+            // limit is meaningless for processes, as we just use a local path.
```

Comment:
We don't bother explaining this for `cpu_limit` or `memory_limit`. Think we should document it for none of them or all of them.

```suggestion
```